[next] useFBO rewrite (#1207)
threlte/threlte: apps/docs/src/content/reference/extras/use-fbo.mdx

type: 'hook'
---

`useFBO` (Framebuffer Object) is a port of [drei's useFBO](https://github.com/pmndrs/drei#usefbo) that creates a [THREE.WebGLRenderTarget](https://threejs.org/docs/index.html#api/en/renderers/WebGLRenderTarget).

> Framebuffer objects are useful when you want to render a scene but not display it. For example, rendering a scene to a framebuffer then passing that to a fragment shader as a texture.

Reviewer: The wording here is pre-existing but upon re-reading maybe can be improved / simplified? 
```suggestion
Framebuffer objects are useful when you want to render a scene to something other than the canvas, such as a texture.
```

Author: what do you think about this

> Framebuffer objects are useful when you want to render a scene to something other than the canvas. In ThreeJS, you do this by rendering the scene to a `RenderTarget`. The render target has a `texture` property that can be used for various things such as postprocessing effects or applying the texture to something in the scene.
